packages/brite-core/src/plugins/brite-cli-plugin-webpack/common.ts

```diff
@@ -36,6 +36,11 @@ const postcssLoader = {
     loader: 'postcss-loader',
     options: {
         sourceMap: true,
+        plugins: [require('postcss-cssnext')({
+            autoprefixer: {
+                browsers: ['last 2 versions'],
```

Comment:
We should revise this list and centralize it for `autoprefixer` and `babel-preset-env`. 

We support much more than jus the last 2 versions.

Reply:
Yeah, in this PR though? This was mostly just copy pasta.

Reply:
Yea, this was more of a note.